include/engine/datafacade/internal_datafacade.hpp

```diff
@@ -76,7 +76,7 @@ class InternalDataFacade final : public BaseDataFacade
 
     util::ShM<util::Coordinate, false>::vector m_coordinate_list;
     util::PackedVector<OSMNodeID, false> m_osmnodeid_list;
-    util::ShM<NodeID, false>::vector m_via_node_list;
+    util::ShM<extractor::GeometryID, false>::vector m_via_node_list;
```

Comment:
why is a a node now referred to by a geometry id? Isn't the list still technically holding nodes?